wiki/People/Staff_Log/2019/en.md

```diff
@@ -4,11 +4,58 @@
 
 - Moved [mancuso_JM_](https://osu.ppy.sh/users/521568) to the osu! Alumni (2019-01-31)
 
-### BN removals
+### Beatmap Nominator removals
 
 - [Trynna](https://osu.ppy.sh/users/2652951) - resignation (2019-01-04)
 - [Sandalphon](https://osu.ppy.sh/users/2633753) - resignation (2019-01-13)
 - [Natsu](https://osu.ppy.sh/users/1953876) - inactivity (2019-01-22)
 - [DTM9 Nowa](https://osu.ppy.sh/users/5428909) - failing probation (2019-01-22)
 - [Net0](https://osu.ppy.sh/users/5099768) - inappropriate behaviour (2019-01-25)
 - [iYiyo](https://osu.ppy.sh/users/3919785) - resignation (2019-01-30)
+
+## February
+
+- Removed [Yuii-](https://osu.ppy.sh/users/2935923) from the Quality Assurance Team (2019-02-16)
+
+### Beatmap Nominator removals
+
+- Removed [Garden](https://osu.ppy.sh/users/2849992) - resignation (2019-02-03)
+- Removed [Sonnyc](https://osu.ppy.sh/users/11771) - failing probation (2019-02-05)
+
+### Beatmap Nominator additions
+
+- [Nevo](https://osu.ppy.sh/users/7451883) (2019-02-16)
```

Comment:
All the additions here happen on the same day, so it should have a `All users listed below were added on 2019-02-16.` heading.